Data-tranformations/fairgenomes/analyses.yarrrml.yaml

```diff
@@ -0,0 +1,26 @@
+prefixes:
+  ega.dataset: https://www.ebi.ac.uk/ega/datasets/
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  OMIABIS: http://purl.obolibrary.org/obo/OMIABIS_
+  OBI: http://purl.obolibrary.org/obo/OBI_
+  ARF: http://purl.allotrope.org/ontologies/result# #this is new and it ends with a #
+  EDAM:http://edamontology.org/ 
+  FIX: http://purl.obolibrary.org/obo/FIX_
+  GENEPIO: http://purl.obolibrary.org/obo/GENEPIO_
+
+mappings:
+  fairgenomes:
+    sources:
+      - [analyses.csv~csv]
+    s: ega.dataset:$(accession_id)
+    po:
+      - [a, EDAM:operation_2945] #this refers to analysis from the model
+      - [OMIABIS:0000006, ega.dataset:$(accession_id)~iri] # I am unsure about this
```

Comment:
Se also comment for samples. In any case `ega.dataset` should be replaced with `ega.analyses` or another appropriate namespace.